### ISSUE TITLE
Incompatible with jwt-go 3.0.0

### ISSUE THREAD

**Issue by mgerb** [go-chi/jwtauth#18]
```
vendor/github.com/go-chi/jwtauth/jwtauth.go:44: parser.SkipClaimsValidation undefined (type *jwt.Parser has no field or method SkipClaimsValidation)
```

Looks like SkipClaimsValidation is no longer part of the Parser struct in 3.0.0.
https://github.com/dgrijalva/jwt-go/blob/release_3_0_0/parser.go

Using dep to install dependencies it tries to pull in version 3.0.0 because that is what is listed in the Gopkg.toml file.

```
[[constraint]]
  name = "github.com/dgrijalva/jwt-go"
  version = "^3.0.0"
```

**Comment by VojtechVitek** [comment]
Blocked by upstream repo: https://github.com/dgrijalva/jwt-go/issues/221 (see https://github.com/go-chi/jwtauth/commit/6075ccf8678cf2c941c7e816937c8ebf93a0b25f).

You'll want to use direct SHAs until they release v3.0.1+:
```
[[constraint]]
  name = "github.com/go-chi/jwtauth"
  revision = "e701090bf80b87b3f81b27edf0559ab044b11181"
# TODO: See https://github.com/go-chi/jwtauth/commit/6075ccf8678cf2c941c7e816937c8ebf93a0b25f
# version = "^3.0.1"

[[constraint]]
  name = "github.com/dgrijalva/jwt-go"
  revision = "a539ee1a749a2b895533f979515ac7e6e0f5b650"
# TODO: See https://github.com/dgrijalva/jwt-go/issues/221
# version = "^v3.0.0"
```

**Comment by explodingcamera** [comment]
Also if you don't import `jwt-go` directly you need to add
```toml
required = ["github.com/dgrijalva/jwt-go"]
```

**Comment by dgrijalva** [comment]
I just released 3.1.0, which includes the `SkipClaimsValidation` option. Hope that helps.

**Comment by VojtechVitek** [comment]
Fixed in 34cf10b6ac177a2112da93be5c0aac45c100c448.

v3.1.1 released https://github.com/go-chi/jwtauth/releases/tag/v3.1.1
